pkg/chrootarchive/cgroup_unix.go

```diff
@@ -0,0 +1,29 @@
+//+build !windows
+
+package chrootarchive
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/containerd/cgroups"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const MEMORY_LIMIT = 20 * 1024 * 1024 // 20MB
```

Comment:
Any reason for choosing 20MB specifically?

Reply:
I found this number to perform well in my tests